assets/scss/layouts/header/_header.scss

```diff
@@ -65,11 +68,14 @@
     > a {
```

Comment:
Can we replace tag selector on class selector? I know that is not part of your ticket, but if it is fast - will be great

Reply:
Done